tests/integration/test_match_huggingface.py

```diff
@@ -0,0 +1,17 @@
+import torch
+from transformers import AutoModelForCausalLM
+
+from transformer_lens import HookedTransformer
+
+
+class TestMatchHuggingFace:
+    def test_test_mlp_outputs_match(self):
+        tl_model = HookedTransformer.from_pretrained_no_processing("gpt2", device="cpu")
+        hf_model = AutoModelForCausalLM.from_pretrained("gpt2", device_map="cpu")
+        test_tensor = torch.randn((1, 1, tl_model.cfg.d_model))
```

Comment:
Can you turn the model name into a parametrized fixture? That will make it really easy for us to expand this for multiple models without rewriting the test.